pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -2,25 +2,25 @@
 
 = The {ProductShortName} configuration file
 
-You need to create a configuration file, named `pantheon2.yml`, in the root directory of your repository, to enable a Git repository as a source that {ProductShortName} can use to import content. This yaml file provides necessary information that {ProductShortName} uses to interact with files in the repository.
+You must create a configuration file, named `pantheon2.yml`, in the root directory of your repository. You use this file to designate a Git repository as a source that {ProductShortName} can use to import content. This configuration file provides the information necessary for {ProductShortName} to interact with files in the repository.
 
-The configuration file enables the following:
+The configuration file allows you to perform the following actions:
```

Comment:
One of the first tech writing guidelines I learned was: Don't use "allows", Do use "enables".

"Enables" is more active and sounds more confident and purposeful than "allows".

I'd suggest: `enables the following actions:`

And if you go with this suggestion, then I think that the actions should start with gerunds (importing, uploading).

Reply:
Agreed re. 'enables' - the only question I'd raise here is that if we remove 'you to perform', it becomes unclear who is doing what. Pantheon doesn't do this; the user does, and this is the vehicle that makes it possible for the user to do so.

Would you have any issues with keeping that in?

Reply:
No, that's fine, I see what you mean.